CONTRIBUTING.md

```diff
@@ -26,6 +27,99 @@ The easier it is for us to reproduce it, the faster it'll be fixed!
 
 Please don't include any private/sensitive information in your issue!
 
+## Contributing to Libpod
+
+This section describes how to start a contribution to Libpod.
+
+### Prepare your environment
+
+Read the [install documentation to see how to install dependencies](install.md) .
+
+The install documentation will illustrate the following steps::
+- install libs and tools
+- check installed versions
+- configure network
+- how to install libpod from sources
+
+### Fork and clone libpod
+
+First you need to fork this project on GitHub.
+
+First go to your `$GOPATH` `cd $GOPATH/src`.
+
+Create a path that correspond to your clone `mkdir -p github.com/<you>`.
+
+Clone your fork locally:
+```shell
+$ git clone git@github.com:<you>/libpod github.com/<you>
+$ cd github.com/<you>/libpod
+```
```

Comment:
Could this be simplified with just a single step:  ``git clone git@github.com:<github username>/libpod $GOPATH/github.com/containers/libpod``?  That will create the directory and error if it already exists.

Reply:
yeah, preferring use:
`git clone git@github.com:<you>/libpod github.com/<you>$GOPATH/github.com/<you>/libpod ` for a more closer golang approach similar `go get` to behaviour .

I also added in my latest changes a `go get` example and a `$GOPATH` warning to users that need to configure it.

Thanks for comments :)